### ISSUE TITLE
ServerHangWatchdog detected that a single server tick took 60.00 seconds

### ISSUE THREAD

**Issue by trunksbomb** [florensie/swimmies#9]
https://pastebin.com/sbchfZHC

```
-- Head --
Thread: Server Watchdog
Suspected Mod:
ExpandAbility (expandability), Version: 9.0.0
Issue tracker URL: https://github.com/florensie/expandability/issues
Mixin class: be.florens.expandability.mixin.fluidcollision.EntityMixin
Target: net.minecraft.world.entity.Entity
at TRANSFORMER/minecraft@1.20.1/net.minecraft.world.entity.Entity.findFluidDistances(Entity.java:16146)
```

If I had to guess, I'd imagine it's getting stuck in an infinite loop in the for loop in this method call:

https://github.com/florensie/ExpandAbility/blob/76b213073b3558a44dccd9e74c9107f4469a8d1a/common/src/main/java/be/florens/expandability/mixin/fluidcollision/EntityMixin.java#L73

Forge 47.2.20
Minecraft 1.20.1

This isn't my crash log; it was submitted to the Valhelsia modpack Discord in the Support channel.

**Comment by florensie** [comment]
Thanks for the report. I've received one other similar report, and I'm currently re-doing the misbehaving code.

**Comment by florensie** [comment]
Fixed in 2b589c3eace1e6ec4ca336c84fd36836016c0625, I should be able to push out a release soon.
